<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QWidget" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0">
    <widget class="QStatusBar" name="statusBar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupBox000">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="4" column="0">
        <widget class="QLineEdit" name="PortlineEdit2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>8080</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QTextBrowser" name="textBrowser">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>30</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="IPpushButton2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>测试6</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>节点1（温湿度）</string>
            </property>
            <widget class="QWidget" name="layoutWidget_2">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>121</width>
               <height>31</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="gBlabel">
                <property name="font">
                 <font>
                  <pointsize>9</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>在线状态：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QLabel" name="gBlabel3">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>70</y>
               <width>71</width>
               <height>21</height>
              </rect>
             </property>
             <property name="text">
              <string>传感器数据：</string>
             </property>
            </widget>
            <widget class="QWidget" name="layoutWidget_3">
             <property name="geometry">
              <rect>
               <x>170</x>
               <y>20</y>
               <width>141</width>
               <height>31</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_1">
              <item>
               <widget class="QLabel" name="gBlabel2">
                <property name="text">
                 <string>电池电量：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="gBlabel2_2">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="gBlabel2_3">
                <property name="text">
                 <string>%</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox2">
            <property name="title">
             <string>节点2（光照强度）</string>
            </property>
            <widget class="QWidget" name="layoutWidget_4">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>121</width>
               <height>31</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout2">
              <item>
               <widget class="QLabel" name="gB2label">
                <property name="text">
                 <string>在线状态：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame2">
                <property name="frameShape">
                 <enum>QFrame::Box</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="layoutWidget_5">
             <property name="geometry">
              <rect>
               <x>170</x>
               <y>20</y>
               <width>141</width>
               <height>31</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout2_2">
              <item>
               <widget class="QLabel" name="gB2label2">
                <property name="text">
                 <string>电池电量：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="gB2label2_2">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="gB2label2_3">
                <property name="text">
                 <string>%</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QLabel" name="gB2label3">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>70</y>
               <width>71</width>
               <height>21</height>
              </rect>
             </property>
             <property name="text">
              <string>传感器数据：</string>
             </property>
            </widget>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QGroupBox" name="groupBox3">
            <property name="title">
             <string>节点3（气体浓度）</string>
            </property>
            <widget class="QWidget" name="layoutWidget_6">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>121</width>
               <height>31</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_31">
              <item>
               <widget class="QLabel" name="gB3label">
                <property name="text">
                 <string>在线状态：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame3">
                <property name="frameShape">
                 <enum>QFrame::Box</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="layoutWidget_7">
             <property name="geometry">
              <rect>
               <x>170</x>
               <y>20</y>
               <width>141</width>
               <height>31</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_61">
              <item>
               <widget class="QLabel" name="gB3label2">
                <property name="text">
                 <string>电池电量：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="gB3label2_2">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="gB3label2_3">
                <property name="text">
                 <string>%</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QLabel" name="gB2label3_2">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>70</y>
               <width>71</width>
               <height>21</height>
              </rect>
             </property>
             <property name="text">
              <string>传感器数据：</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="3">
          <widget class="QPushButton" name="pushButton4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>设置</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pushButton3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>清空数据</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>启动服务器</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>保存</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="IPpushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>测试5</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>常用传感节点</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QSplitter" name="tab_splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QChartView" name="graphicsView"/>
          <widget class="QChartView" name="graphicsView2"/>
          <widget class="QChartView" name="graphicsView3"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>电机传感节点</string>
       </attribute>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
